buildspec/release.yml

```diff
@@ -0,0 +1,46 @@
+version: 0.2
+
+# Publishes the specified branch to the Visual Studio Marketplace, and archives the vsix package to S3
+env:
+  variables:
+    GIT_REPO: "https://github.com/aws/aws-toolkit-vscode.git"
+    GIT_BRANCH: "master"
+    STAGE: "BETA"
+  parameter-store:
+    MARKETPLACE_PERSONAL_ACCESS_TOKEN: "/marketplace/personal-access-token/toolkit-vscode"
+
+phases:
+  install:
+    commands:
+      - New-Item -Path . -Name "artifacts" -ItemType Directory
+      - git clone $Env:GIT_REPO aws-toolkit-vscode
+  pre_build:
+    commands:
+      - cd aws-toolkit-vscode
+      - git checkout $Env:GIT_BRANCH
+      - npm install
+  build:
+    commands:
+      - ./node_modules/.bin/vsce package
+      # The generated file is dynamically named.
+      # We could package to a static filename, but it is better to archive a file that contains the version in it.
+      - $vsixFiles = Get-ChildItem -Filter aws-toolkit-vscode*.vsix
+      # Assumption: $vsixFiles is always one file
```

Comment:
Should we assert this assumption?

Reply:
Alternatively, we could pull the version number from `package.json`, since that should be the single-source-of-truth. This would protect against accidentally publishing an older version.